stylesheets/_global-wrapper.scss

```diff
@@ -185,23 +185,27 @@ ul.footer-nav {
   }
 }
 
-ul.social-nav {
+ul#social-nav {
```

Comment:
We try to stay away from using IDs to style elements in our CSS - can you switch this back to a class?

IDs can only be used a single time on the page, by using classes it allows us to reuse elements if we need to.